### DISCUSSION TITLE
Error message when applying to "student pack"

### DISCUSSION THREAD

**Discussion by RaniaRez** [community/community#55472] [GitHub Education]
Hey!

I am a computer science student and I am currently experiencing issues while trying to apply for the GitHub Student Pack. I have tried several times to apply, but I am still encountering errors and I am unable to complete the application process.

I have included my student card as proof of my student status. Despite my efforts, I am still unable to proceed with the application.

I would greatly appreciate any assistance you can provide in resolving this issue so that I can take advantage of the resources provided by the GitHub Student Pack. Please let me know if there are any additional steps I can take to resolve this issue.

Thanks!

**Comment by jge162** [comment]
instead of school ID have you tried your school transcripts showing your current term? that worked for me and a bunch of people here.

**Comment by samus-aran** [comment]
Hi @RaniaRez 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and I'll be happy to assist!
